tasks/deploy.js

```diff
@@ -0,0 +1,587 @@
+    ...contractImports,
+    contractConfigs,
+    deployFunction,
+    maxAmount: getOptionalEnvVar("MAX_AMOUNT", maxAmount),
+    unitPrice: toWei("0.1"),
+    nbUnits: toBN("100000"),
+    maxUnits: getOptionalEnvVar("MAX_UNITS", maxUnits),
+    tokenAddr: ETH_TOKEN,
+    erc20TokenName: getEnvVar("ERC20_TOKEN_NAME"),
+    erc20TokenSymbol: getEnvVar("ERC20_TOKEN_SYMBOL"),
+    erc20TokenDecimals: getEnvVar("ERC20_TOKEN_DECIMALS"),
+    erc20TokenAddress: UNITS,
+    maxChunks: getOptionalEnvVar("MAX_CHUNKS", maximumChunks),
+    votingPeriod: getOptionalEnvVar("VOTING_PERIOD_SECONDS", 600), // 600 secs = 10 min
+    gracePeriod: getOptionalEnvVar("GRACE_PERIOD_SECONDS", 600), // 600 secs = 10 min
+    offchainVoting: true,
+    finalize: false,
+    maxExternalTokens: 100,
+    couponCreatorAddress: getOptionalEnvVar(
+      "COUPON_CREATOR_ADDR",
+      getEnvVar("DAO_OWNER_ADDR")
+    ),
+    kycSignerAddress: getOptionalEnvVar(
+      "KYC_SIGNER_ADDR",
+      getEnvVar("DAO_OWNER_ADDR")
+    ),
+    kycMaxMembers: getOptionalEnvVar("KYC_MAX_MEMBERS", toBN(1000)),
+    kycFundTargetAddress: getOptionalEnvVar(
+      "KYC_MULTISIG_FUND_ADDR",
+      ZERO_ADDRESS
+    ),
+    daoName: getEnvVar("DAO_NAME"),
+    owner: getEnvVar("DAO_OWNER_ADDR"),
+    offchainAdmin: getOptionalEnvVar(
+      "OFFCHAIN_ADMIN_ADDR",
+      getEnvVar("DAO_OWNER_ADDR")
+    ),
+    deployTestTokens: true,
+    supplyTestToken1: 1000000,
+    supplyTestToken2: 1000000,
+    supplyPixelNFT: 100,
+    supplyOLToken: toBN("1000000000000000000000000"),
+    erc1155TestTokenUri: "1155 test token",
+    gasPriceLimit: getOptionalEnvVar("GAS_PRICE_LIMIT", 0 /* disabled */),
+    spendLimitPeriod: getOptionalEnvVar("SPEND_LIMIT_PERIOD", 0 /* disabled */),
+    spendLimitEth: getOptionalEnvVar("SPEND_LIMIT_ETH", 0 /* disabled */),
+    gelato: getOptionalEnvVar("GELATO_ADDR", ZERO_ADDRESS),
+    weth: weth.address,
+    maintainerTokenAddress: getOptionalEnvVar("MAINTAINER_TOKEN_ADDR", UNITS),
+  });
+};
+
+const deploy = async (opts) => {
+  const action = DeploymentActions[opts.network];
+  if (action) {
+    return action(opts);
+  }
+  throw new Error(`Unsupported operation ${opts.network}`);
+};
+
+const getOrCreateDaoArtifacts = async (hre, hardHatImports) => {
+  const DaoArtifacts = hardHatImports.DaoArtifacts;
+  let daoArtifacts;
+  const factory = await hre.ethers.getContractFactory(
+    DaoArtifacts.contractName
+  );
+  if (process.env.DAO_ARTIFACTS_CONTRACT_ADDR) {
+    log(`Attach to existing DaoArtifacts contract`);
+    daoArtifacts = await factory.attach(
+      process.env.DAO_ARTIFACTS_CONTRACT_ADDR
+    );
+  } else {
+    log(`Creating new DaoArtifacts contract`);
+    const daoArtifact = await factory.deploy();
+    daoArtifacts = await daoArtifact.deployed();
+  }
+  log(`DaoArtifacts: ${daoArtifacts.address}`);
+  return daoArtifacts;
+};
+
+const getEnvVar = (name) => {
+  if (!process.env[name]) throw Error(`Missing env var: ${name}`);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819145)
```suggestion
  if (!process.env[name]) {throw Error(`Missing env var: ${name}`);}
```